### PR TITLE
Refactoring: remove copy-paste

### PR DESCRIPTION
**Pull request by 4ertus2**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Improvement

Short description (up to few sentences):
Remove some copy-paste (TemporaryFile and TemporaryFileStream)

### CODE REVIEW

File: dbms/src/Common/filesystemHelpers.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#include <common/config_common.h>
+#include <Poco/File.h>
+#include <Poco/Path.h>
+
+namespace DB
+{
+
+bool checkFreeSpace(const std::string & path, size_t data_size)
+{
+#if !UNBUNDLED
```
Comment (abyss7) [inline dbms/src/Common/filesystemHelpers.cpp]: Why can't we get free space if we're built in unbundled mode?

File: dbms/src/Common/filesystemHelpers.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#include <Common/filesystemHelpers.h>
+#include <common/config_common.h>
+#include <Poco/File.h>
+#include <Poco/Path.h>
+
+namespace DB
+{
+
+bool checkFreeSpace(const std::string & path, size_t data_size)
```
Comment (KochetovNicolai) [inline dbms/src/Common/filesystemHelpers.cpp]: Not very good name. Can't understand that it's connected with files.
Maybe `hasEnoughSpaceInDirectory(path, required_bytes)`?

Comment (KochetovNicolai) [reply dbms/src/Common/filesystemHelpers.cpp]: Maybe add namespace?

**Review by KochetovNicolai — APPROVED** [review]

File: dbms/src/Common/filesystemHelpers.cpp (hunk 1 above)
Comment (4ertus2) [reply dbms/src/Common/filesystemHelpers.cpp]: I don't know. It's not related to this patch. It just keep logic as is.

File: dbms/src/Common/filesystemHelpers.cpp (hunk 2 above)
Comment (4ertus2) [reply dbms/src/Common/filesystemHelpers.cpp]: The fucntion's been renamed. I've thought about a namespace. I think it's better to make it for removing explicit Poco::File usage and then append these functions into it.

**Pull request merged**